apps/onnx/Makefile

```diff
@@ -76,11 +76,11 @@ test: $(BIN)/$(HL_TARGET)/onnx_converter_test $(BIN)/$(HL_TARGET)/onnx_converter
 	LD_LIBRARY_PATH=$(BIN) $(BIN)/$(HL_TARGET)/onnx_converter_test
 	LD_LIBRARY_PATH=$(BIN) $(BIN)/$(HL_TARGET)/onnx_converter_generator_test
 
-PYTHON ?= python3
-PYBIND11_CFLAGS = $(shell $(PYTHON) -m pybind11 --includes) -frtti
+PYTHON ?= python
```

Comment:
Are we going to support python2 at this point in time?

Reply:
There is no point since it's going to be supported only for a semester or so. I reverted to python3